content/applications/finance/accounting/taxation/taxes/vat_units.rst

```diff
@@ -0,0 +1,86 @@
+=========
+VAT units
+=========
+
+.. important::
+   This is only applicable to multi-company environments.
+
+A **VAT unit** is a group of VAT-taxable enterprises that are legally independent of each other but
+are closely linked financially, organizationally, and economically and therefore considered the same
+VAT-taxable enterprise. **VAT units** are not mandatory, but if created, constituent companies of
+the unit must belong to the same **country**, use the same **currency**, and one company must be
+designated as the **representative** company of the **VAT unit**. **VAT units** receive a specific
+**tax ID** intended only for **tax returns**. **Constituent** companies keep their **tax ID** used
+for **commercial purposes**.
+
+.. example::
+   Enterprise **A** owes 300.000,00€ of VAT taxes and enterprise **B** can recover 280.000,00€ of
+   VAT taxes. They form up as a **VAT unit** so that the two amounts balance out and must conjointly
+   only pay 20.000,00€ of VAT taxes.
```

Comment:
In English, the euro symbol goes before the amount. In other languages, it's often after the amount, with a space in between.
```suggestion
   Enterprise **A** owes €300.000,00 of VAT taxes and enterprise **B** can recover €280.000,00 of
   VAT taxes. They form up as a **VAT unit** so that the two amounts balance out and must conjointly
   only pay €20.000,00 of VAT taxes.
```